lib/sinon/stub-descriptor.js

```diff
@@ -0,0 +1,43 @@
+"use strict";
+
+var deprecated = require("./util/core/deprecated");
+var objectKeys = require("./util/core/object-keys");
+var spy = require("./spy");
+var wrapMethod = require("./util/core/wrap-method");
+
+// This is deprecated and will be removed in a future version of sinon.
+// We will only consider pull requests that fix serious bugs in the implementation
+function stubDescriptor(object, property, descriptor) {
+    var wrapper;
+
+    deprecated.printWarning(
+      "sinon.stub(obj, 'meth', fn) is deprecated and will be removed from" +
```

Comment:
Hi, friends, I noticed that the third argument for the stub function is no longer going to be valid. While this seems to be okay when stubbing functions I'd like to know how could we stub the `get/set` properties of descriptors.

Are we going to have a specific API for that or users should just use `Object.getOwnPropertyDescriptor` and then stub the `get` and `set` functions in there?

Also, if you want to take a look at how it works right now, #692 is the PR responsible for introducing it.